### PR TITLE
Template elements are no more mandatory in final name.

### LINKED ISSUE #344: Ability to set a custom name without templates restriction
OS: Ubuntu 20.04

Launch command used:
./destreamer.sh --format mp4 -o out -i "my_url" -t "Lesson 5"

Hi there,

I'm not able to set a custom template name like the one above. I must set at least one of the "magic sequences" to make it work.

For example:
"Lesson 5" -> This is not a valid title template. Thus, destreamer refuses to use my custom file name and simply fallbacks to the default one.

"Lesson 5 {author}" -> This one is a valid template. Destreamer accepts my custom title template.

I don't understand this behaviour. Why I can't set a custom name?

### PR DESCRIPTION
**Pull request by goldmont**
[Closed #344](https://github.com/snobu/destreamer/issues/344)

### CODE REVIEW

**Review by lukaarma — APPROVED** [review]
I didn't account that this could be used as "custom name output". All seems good to me

**Review by lukaarma — CHANGES_REQUESTED** [review]
@goldmont could you update the README section of the title template to reflect this change? Let everyone know that no "magic sequence" is required and this could be used as as output name?
If not I'll do it when I can and I'll merge it

**Comment by goldmont** [comment]
Ok man, let me do it

**Comment by goldmont** [comment]
I'm done @lukaarma

**Comment by snobu** [comment]
@lukaarma when this meets your quality bar just merge it, don't wait for me.

File: README.md [hunk 1]
```diff
@@ -197,7 +197,15 @@ You can use one or more of the following magic sequence which will get substitut
...
 
-Example -
+Examples -
+```
+Input:
+    -t 'This is an example'
+
+Expected filename:
+    This is an example.mkv
+```
```
Comment (lukaarma) [inline README.md:207]: Please merge this code brackets with the other example, add another mixed example like 

-t 'This is an example by {author}'

File: README.md [hunk 2]
```diff
@@ -185,7 +185,7 @@ https://web.microsoftstream.com/video/xxxxxxxx-aaaa-xxxx-xxxx-xxxxxxxxxxxx
 ```
 
 ### Title template
-The `-t` option allows users to input a template string for the output file names.
+The `-t` option allows users to input a custom or a template string for the output file names.
```
Comment (lukaarma) [inline README.md:188]: I'm having trouble understanding this edit, could you please rephrase it?

My suggestion would be like 'The `-t` option allows user to specify a custom filename for the videos.'

**Review by lukaarma — CHANGES_REQUESTED** [review]

**Comment by lukaarma** [comment]
@snobu there is a minor linting error, should I revert it or its not worth for such a small fix?

**Comment by goldmont** [comment]
@lukaarma Thank you, keep up the good work.

**Comment by goldmont** [comment]
> @snobu there is a minor linting error, should I revert it or its not worth for such a small fix?

Where?

**Comment by lukaarma** [comment]
```
destreamer/destreamer/src/CommandLineParser.ts
Error:   177:9  error  'finalTemplate' is never reassigned. Use 'const' instead  prefer-const
```

**Comment by goldmont** [comment]
I'm gonna fix it right now.

**Pull request merged**